src/main/java/com/rtb/projectmanagementtool/private-comments/PrivateCommentController.java

```diff
@@ -0,0 +1,102 @@
+package com.rtb.projectmanagementtool.privatecomment;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.rtb.projectmanagementtool.user.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Class controlling the CommentData object. */
+public final class PrivateCommentController {
+
+  private DatastoreService datastore;
+  private static final Filter NO_QUERY_FILTER = null;
+  private static final int NO_QUERY_LIMIT = Integer.MAX_VALUE;
+  private static final SortPredicate NO_QUERY_SORT = null;
+
+  public PrivateCommentController(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  public Long addPrivateComment(PrivateCommentData privateComment) {
+    Entity entity = privateComment.toEntity();
+    Key key = datastore.put(entity);
+    long commentID = key.getId();
+    privateComment.setCommentID(commentID);
+    return commentID;
+  }
```

Comment:
Would this be able to add multiple private comments for one userID/taskID combo?

Reply:
I was thinking we'd only have 1 private comment per task (since there will only be 1 comment box) but it can constantly be edited and updated.